app/views/candidate_interface/guidance/index.html.erb

```diff
@@ -0,0 +1,80 @@
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.apply_opens.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">Start applying for courses. You can apply to 4 courses at a time. Courses can fill up quickly, so you should apply as soon as you can.<br><br>You should make sure each course you choose is offered by a different training provider so you can have more chance of success.<br><br>Training providers will have 40 working days to decide to offer you a place on a course. Holiday periods in England over December and March or April are not included in the 40 working days. Each course you apply to will show the date for when providers need to respond by.<br><br>If training providers do not respond in time, your application will be unsuccessful.<br><br>You can apply to 4 courses again if all your applications are unsuccessful.<br><br>You’ll have 10 working days to respond to any offers you receive. If you do not respond, the offer will be automatically declined.</td>
+          </tr><tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.show_summer_recruitment_banner.to_s(:govuk_date) %></td>
+            <td class="govuk-table__cell">The time training providers have to make a decision about your application is reduced from 40 working days to 20 working days.</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.apply_1_deadline.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">If you’ve not already applied for teacher training, you will not be able to apply after <%= CycleTimetable.apply_1_deadline.to_s(:govuk_date) %>.<br><br> If you’ve already applied, but your applications were unsuccessful, you can continue to apply for courses until <%= CycleTimetable.apply_2_deadline.to_s(:govuk_date_and_time) %>.</td>
+          </tr>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell"><%= CycleTimetable.reject_by_default.to_s(:govuk_date_and_time) %></td>
+            <td class="govuk-table__cell">The last day for training providers to make a decision on all applications for courses starting in September <%= @recruitment_cycle_year %>.</td>
+          </tr>
+        </tbody>
+      </table>
+    <% else %>
+      <h1 class="govuk-heading-l">The application process for courses starting in September <%= @recruitment_cycle_year %></h1>
+      <p class="govuk-body">For courses starting in September <%= @recruitment_cycle_year %>, you can start applying a year before they start in October <%= @recruitment_cycle_year - 1 %>.</p>
+      <p class="govuk-body">Most teacher training courses are one year and start in September when the academic year in England starts. Part time courses are usually 2 years.</p>
+
+      <br>
+
+      <table class="govuk-table">
```

Comment:
Could consider using the [`govuk_table` helper](https://govuk-components.netlify.app/components/table) here, to avoid having to add all the class names manually?